piet-gpu-hal/Cargo.toml

```
@@ -7,12 +7,12 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [dependencies]
-ash = "0.33"
-ash-window = "0.7"
-raw-window-handle = "0.3"
-bitflags = "1.2.1"
-smallvec = "1.6.1"
-bytemuck = "1.7.2"
+ash = "0.37"
```

```suggestion
ash = {version = "0.37", features=["linked"]}
```
current behavior is now opt-in

---

I don't think we depend on the "linked" behavior. Perhaps you can explain why you think we do?

---

> I don't think we depend on the "linked" behavior. Perhaps you can explain why you think we do?

I don't know your internals but 
Entry::new()?
was renamed
Entry::linked()
and hidden behind this flag